aerospike/tests/common.py

```diff
@@ -8,7 +8,7 @@
 HERE = os.path.dirname(os.path.abspath(__file__))
 COMPOSE_FILE = os.path.join(HERE, 'docker', 'docker-compose.yaml')
 HOST = get_docker_hostname()
-PORT = 3003
+PORT = 3000
```

Comment:
why did you change this?

Reply:
3003 was for a direct socket connect which we no longer do

Reply:
okay